backend/app/tests/services/schedule/test_get_schedule_service.py

```diff
@@ -0,0 +1,41 @@
+from sqlmodel import Session
+
+from app.services.schedule import get_schedule
+from app.tests.utils.schedule_utils import (
+    create_first_schedule,
+    create_second_schedule,
+    first_schedule_string,
+    second_schedule_string,
+)
+
+
+def test_get_schedule_service_when_schedule_does_exist_should_return_none(
+    db: Session,
+) -> None:
+    result = get_schedule(session=db)
+    assert result is None
+
+
+def test_get_schedule_service_when_schedule_exists_should_return_schedule_entity(
+    db: Session,
+) -> None:
+    create_first_schedule(db)
```

Comment:
Should we avoid calling other services when testing a service?

Reply:
Got it. I just wanted to point out the separation of unit tests - if we modify `create_schedule` service in future and it fails, this tests will also fail, making it bit difficult to identify the cause.

Reply:
You are definitely correct about the separation of concerns.  In this case, I thought it would be okay if both `get` and `create` failed if there was only an error with `create`

Reply:
Thinkin about it, having a fixture function that does exactly same thing as the service function might conflict with DRY principle. Because if you modify the service, you also have to modify the fixture. So I guess there's some trade-offs of each.